File: URI/1 - INICIANTE/C/1131 - Grenais.c
#include <stdio.h>

int main()
{
	int g1, g2, w = 0, inter = 0, grenal = 0, gremio = 0, empates = 0, op;
	
	while(w == 0)
	{
		grenal += 1;
		scanf(" %d %d", &g1, &g2);
		if(g1 == g2)
		{
			empates +=1;
		}else if (g1 > g2){
			inter += 1;
		}else{
			gremio += 1;
		}
		printf("Novo grenal (1-sim 2-nao)\n");
		scanf(" %d", &op);
		if(op == 2)
		{
			w = 1;
		}
	}
	printf("%d grenais\n", grenal);
	printf("Inter:%d\n", inter);
	printf("Gremio:%d\n", gremio);
	printf("Empates:%d\n", empates);
	if(inter > gremio){
		printf("Inter venceu mais\n");
	}else{
		printf("Gremio venceu mais\n");
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1131 - Grenais.py
w = grenal = inter = gremio = empates = 0
while w == 0:
    grenal += 1
    g1, g2 = input().split(' ')
    g1 = int(g1)
    g2 = int(g2)
    if g1 == g2:
        empates += 1
    elif g1 > g2:
        inter += 1
    else:
        gremio += 1
    print('Novo grenal (1-sim 2-nao)')
    op = int(input())
    if op == 2:
        w = 1
print('{} grenais'.format(grenal))
print('Inter:{}'.format(inter))
print('Gremio:{}'.format(gremio))
print('Empates:{}'.format(empates))
if(inter > gremio):
    print('Inter venceu mais')
else:
    print('Gremio venceu mais')